api/iode.h

```diff
@@ -127,8 +124,8 @@
 #define KOBJS(kdb)   ((kdb)->k_objs)
 
 #define KONAME(kdb, pos)    ((kdb)->k_objs[pos].o_name)                 // name of the object
-#define KSOVAL(kdb, pos)    ((kdb)->k_objs[pos].o_val)                  // position of the object in the memory
-#define KGOVAL(kdb, pos)    (SW_getptr((kdb)->k_objs[pos].o_val))       // pointer to the object (as a char*)
+#define KSOVAL(kdb, pos)    ((kdb)->k_objs[pos].o_val)                  // handle of the object in the scr4/swap memory
```

Comment:
What the difference between handle and position ?

Reply:
A handle is a way to identify a chunck of memory in the scr4/swap library. It's more complicated than 
```
ptr = list_of_pointers[handle]
``` 
but as a first approach it is a right representation. Actually you can retrieve the real memory position (the "pointer") by a call to 
```
SW_getptr(handle)
```